modules/siddhi-core/src/main/java/org/wso2/siddhi/core/executor/function/ConvertFunctionExecutor.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c)  2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
  *
```

Comment:
I think we need to keep the date which class was originally written but not  the day it is updated. Refer mail "Re: License Header for WSO2 code"